connectors/src/connectors/slack/lib/workspace_limits.ts

```diff
@@ -206,10 +206,7 @@ export async function isBotAllowed(
   connector: ConnectorResource,
   slackUserInfo: SlackUserInfo
 ): Promise<Result<undefined, Error>> {
-  const displayName = slackUserInfo.display_name ?? "";
-  const realName = slackUserInfo.real_name ?? "";
-
-  const names = [displayName, realName].filter((name) => name.length > 0);
+  const realName = slackUserInfo.real_name;
```

Comment:
Just want to make sure that this isn't a breaking change and that we test before releasing.

Reply:
I talked to the Dust Slack bot using my Slack account and a Slack workflow, both work well 👍